src/link/edit.js

```diff
@@ -175,6 +181,14 @@ const Edit = ( {
 		}
 	};
 
+	const waitOneSecThenDisplayCustomTooltip = () => {
+		setTimeout( () => {
+			setDisplayCustomTooltip( true );
+			// eslint-disable-next-line no-undef
+			localStorage.setItem( 'WikipediaPreviewWordpressPlugin-CustomTooltipDisplayedCount', customTooltipDisplayedCount + 1 );
+		}, 1000 );
```

Comment:
you need to debounce this method, if user close the toolbar within 1 second, this is still called.

Reply:
Good catch! I was able to replicate, bug looked ugly with the popover without the toolbar. I think [adding a guard](https://github.com/wikimedia/wikipediapreview-wordpress/pull/111/commits/1c649545c165341cfc2f77273dbca3129ffa336f) to ensure toolbar is still open should be enough, what do you think? Can you still replicate the bug?